src/dictionary/mod.rs

```diff
@@ -252,35 +254,26 @@ mod tests {
     }
 
     #[test]
-    fn test_serialize_save_read() {
-        let mock = MyFilterMock::new();
+    fn test_serialize_save_read() -> Result<(), Box<dyn Error>> {
+        let path = "./save_read.schema";
+        let expected = Module::new(MyFilterMock::new());
+        expected.save_schema_to_file(path)?;
 
-        let mut serialize = Module::new(mock);
-        let buffer = serialize.serialize(TEXT);
-        if let Err(_) = serialize.save_schema_to_file(PATH) {
-            assert!(false);
-        }
-        let mock1 = MyFilterMock::new();
-        let Ok(new_serializer) = Module::read_schema_from_file(PATH, mock1) else {
-            assert!(false);
-            return;
-        };
+        let actual = Module::read_schema_from_file(path, MyFilterMock::new())?;
```

Comment:
IMO readability is more important in tests than safety. Let me know what you think.